tests/integration/it_locations_test.py

```diff
@@ -0,0 +1,63 @@
+
+from timeless.restaurants.models import Location
+from timeless.companies.models import Company
+
+
+def test_create_location(db_session):
+    """Integration test for adding and selecting Location"""
+    location = create_location(db_session)
+    row = db_session.query(Location).get(location.id)
+    assert row.name == "Krusty Burger"
+
+def test_delete_location(db_session):
+    """Integration test for deleting Location"""
+    location = create_location(db_session)
+    id = location.id
```

Comment:
unused var, let's remove it. also, for future, it's not a good idea use vars like `id`, since python has built-in function with the same name. btw that's why `github` highlights the variable